tests/filecheck/transforms/eqsat-simplify-expressions.mlir

```diff
@@ -0,0 +1,22 @@
+// RUN: xdsl-opt -p eqsat-add-costs %s | filecheck %s
```

Comment:
Should this file be called `eqsat-add-costs.mlir`?

Reply:
Agreed - will update